api/database/migrations/2021_09_17_135548_create_candidate_search_requests_table.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateCandidateSearchRequestsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('candidate_search_requests', function (Blueprint $table) {
+            $table->id();
+            $table->timestamps();
+            $table->string("full_name");
+            $table->string("email");
+            $table->string('department_id')->references('id')->on('departments');
+            $table->string("job_title");
+            $table->text("additional_comments")->nullable();
+            $table->text("pool_candidate_filter_id")->references('id')->on('pool_candidate_filters');
+            $table->date("requested_date");
+            $table->text("status");
```

Comment:
Any reason to use `text` over `string` here?

Reply:
Nope that should be string